### ISSUE TITLE
Live Reality Video freezes when refreshed

### ISSUE THREAD

**Issue by shukieshah** [argonjs/argon#66]
In the iOS app, when I'm in the Live-Reality mode and I try to refresh the view by clicking on the refresh icon, it freezes and the camera no longer provides a live feed.

**Comment by blairmacintyre** [comment]
Can you provide more details?  I haven't seen this.  Is this only in your app?  In all apps?  Are you using the store version of Argon4 or have you tried it on the TestFlight version?

**Comment by shukieshah** [comment]
I don't believe it is specific to my version, another member was having the same issue. My version is the store version (Argon 4) so it may not be the most up to date version available. I have not tried it on the TestFlight version.

**Comment by blairmacintyre** [comment]
Can you provide more specific details:  this happens on which web applications?
